superset/connectors/sqla/models.py

```diff
@@ -830,6 +831,17 @@ def init_on_load(self) -> None:
     def __repr__(self) -> str:
         return str(self.column_name)
 
+    @validates("python_date_format")
+    def validate_pdf_is_iso8601(self, _: str, dt_format: str) -> str:
+        if dt_format in ("epoch_s", "epoch_ms", None):
+            return dt_format
```

Comment:
Mypy should be borking on the return type here as well given `dt_format` could be `None`.